orchagent/portsorch.cpp

```diff
@@ -2968,6 +2974,25 @@ bool PortsOrch::removeLagMember(Port &lag, Port &port)
     LagMemberUpdate update = { lag, port, false };
     notify(SUBJECT_TYPE_LAG_MEMBER_CHANGE, static_cast<void *>(&update));
 
+    // If the number of enabled members is less than the value of this channel miniport,
+    // It will cause that the LAG channel oper_status DOWN,
+    // then we should clear the entire FDB entry.
+    vector<FieldValueTuple> lag_value;
```

Comment:
Why do we need to do this here? there is a `doLagTask `that gets the `oper_status` command and you could invoke the function `removeFDBEntriesByBridgePortID `in that flow.